src/main/resources/application.properties

```diff
@@ -3,27 +3,23 @@ spring.application.name=GabartiScholars-QB-BE
 # Server configuration
 server.port=8080
 
-# Database configuration (will be overridden by environment variables in Docker)
 spring.datasource.url=jdbc:mysql://localhost:3306/mydatabase
 spring.datasource.username=myuser
 spring.datasource.password=secret
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 
-# JPA/Hibernate configuration
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
 spring.jpa.properties.hibernate.format_sql=true
 
-# DevTools configuration for auto-reloading
 spring.devtools.restart.enabled=true
 spring.devtools.livereload.enabled=true
 spring.devtools.restart.poll-interval=1000
 spring.devtools.restart.quiet-period=400
 
-# Disable Spring Boot Docker Compose integration (we're already in Docker)
 spring.docker.compose.enabled=false
 
-# Logging
 logging.level.org.springframework.web=DEBUG
 logging.level.org.hibernate.SQL=DEBUG
+jwt.secret=yourSuperSecretKeyThatIsVeryLongAndSecureNoOneCanGuessThis12345
```

Comment:
Same as above it should be stored in env file at least, and also normally it's generated not written by us and yes definitely it should be very secure and not simple 😆